pkg/ccl/crosscluster/logical/create_logical_replication_stmt.go

```diff
@@ -208,6 +208,8 @@ func createLogicalReplicationStreamPlanHook(
 			defaultConflictResolution = *cr
 		}
 
+		filterRangefeed := options.GetFilterRangefeed()
```

Comment:
nit: I wonder if we should use a more descriptive phase throughout the  code `filterCDCIgnoredTTLDeletes`. This also makes it clear that any logic related to this option is bespoke. @dt wdyt?

Reply:
We definitely could. I chose FilterRangefeed because under the hood, we don't have a choice in what we filter so I wanted the name to be as accurate as possible.